distributed-calculator/README.md

```diff
@@ -616,13 +616,18 @@ The code below shows calls to the "add" and "subtract" services via the Dapr URL
 const daprUrl = `http://localhost:${daprPort}/v1.0/invoke`;
 
 app.post('/calculate/add', async (req, res) => {
-  const addUrl = `${daprUrl}/addapp/method/add`;
-  req.pipe(request(addUrl)).pipe(res);
+  const appUrl = `${daprUrl}/addapp/method/add`;
+  const appResponse = await axios.post(appUrl, req.body);
+  const result = String(appResponse.data);
```

Comment:
Why are we wrapping this in a String? If casting is required you can use `appResponse.data as string` but I am not sure casting is required here?

Reply:
The react client expects a string, and it's not typescript.  This seems like the only way without rerewriting client a bit.